include/miral/miral/window_manager_tools.h

```diff
@@ -194,30 +197,30 @@ class WindowManagerTools
      * \remark the tools hold only a weak_ptr<> to the workspace - there is no need for an explicit "destroy".
      * @return a shared_ptr owning the workspace
      */
-    auto create_workspace() -> std::shared_ptr<Workspace>;
+    auto create_workspace() -> std::shared_ptr<miral::Workspace>;
```

Comment:
Not looked at fixing this, but I'm not a fan of adding noise to code where it isn't necessary (except for deficiencies in this toolchain).